UI/index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Ride My Way</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./assets/css/style.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="./assets/css/mediaQueries.css" />
+    <!-- <script src="main.js"></script> -->
+</head>
+<body>
+    <header class='format-header'>
```

Comment:
Use consistent quotation marks. Either single marks or double quotes for all the html tags.